python/GafferUI/PythonEditor.py

```diff
@@ -71,9 +70,8 @@ def __init__( self, scriptNode, **kw ) :
 			role = GafferUI.MultiLineTextWidget.Role.Code,
 		)
 
-		self.__outputWidget._qtWidget().setObjectName( "gafferPythonEditorOutput" )
-		self.__inputWidget._qtWidget().setObjectName( "gafferPythonEditorInput" )
-
+		self.__outputWidget._qtWidget().setProperty( "gafferTextRole", "output" )
```

Comment:
Let's leave this for later, but perhaps we should be expanding the `MultiLineTextWidget.Role` enum to support this?